src/App.jsx

```diff
@@ -1,33 +1,46 @@
 import React, { useState } from 'react';
+// Components
 import TodoList from './components/TodoList';
-import AddTodoForm from './components/AddTodoForm';
+import NavigationBar from './components/NavigationBar';
+import Footer from './components/Footer';
+// Helpers
+import { scrollToTop } from './helpers/scrollToTop';
+// Third party libraries
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 const App = () => {
   /*
     ============================
     =          STATES          =
     ============================
   */
-  const [newTodo, setNewTodo] = useState("");
+  const [todoList, setTodoList] = useState([]);
 
   /*
     ============================
     =         HANDLERS         =
     ============================
   */
-  const handleAddTodo = (todoTitle) =>{
-    setNewTodo(todoTitle);
+  const handleAddTodo = (newTodo) => {
+    setTodoList((todoList) => [...todoList, newTodo]);
   };
 
   return (
       /* Fragment creation */
       <>
-        <h1>Todo List</h1>
-        <AddTodoForm onAddTodo={handleAddTodo}/>
-        <p>{newTodo}</p>
-        <TodoList/>
+        <header>
+          <NavigationBar onAddTodo={handleAddTodo}/>
+        </header>
+        <main>
+          <TodoList todoList={todoList}/>
+        </main>
+        <Footer/>
+        <a className='button-top' onClick={scrollToTop}>
+          <FontAwesomeIcon icon={["fas", "circle-up"]} size="2x"/>
+        </a>
```

Comment:
That's pretty cool! Looks awesome too! :1st_place_medal:

Reply:
> That's pretty cool! Looks awesome too! 🥇

Thanks!!